### ISSUE TITLE
Mild Pain

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5938]
**Preferred term label:** Mild Pain

**Synonyms**

**Definition (free text, please give PubMed ID)** Pain in the 1-3 range (on a 1-10 scale)

**Parent term (use hpo.jax.org/app)** Pain HP:0012531

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Please use the clinical modifiers for severe,moderate,mild : https://hpo.jax.org/app/browse/term/HP:0012824
